src/libraries/System.Transactions.Local/src/ILLink/ILLink.Descriptors.LibraryBuild.xml

```diff
@@ -0,0 +1,31 @@
+<linker>
+  <assembly fullname="System.Transactions.Local" feature="System.Runtime.InteropServices.BuiltInComInterop.IsSupported" featurevalue="true" featuredefault="true">
```

Comment:
> is there any special guidance on trimming in the context of COM interop?

It isn't supported for the built-in COM interop. It might be prudent to disable trimming on this assembly for now.

Reply:
I just did a cursory look and I think if you set `IsTrimmable` to `false` in the project file it will disable trimming. I defer to @vitek-karas or @MichalStrehovsky if there is another or better way though.

Reply:
Yes, setting IsTrimmable to false should do two things: disable the trimming analyzer at runtime repo build time, and skip injecting the IsTrimmable metadata.

The assembly is still going to be trimmed if someone specifies PublishTrimmed=true in their project (as of .NET 7 trimming only looks at IsTrimmable metadata when doing partial trimming). But the user will get a build time warning that the assembly is not compatible and we should hopefully land in the well-defined "runtime COM interop is disabled and throws a meaningful exception" path at runtime.

Hopefully for .NET 8 we'll get COM source generator, and this can be made trimming friendly with that.

Reply:
Note that we have number of libraries in the repo that set `<IsTrimmable>false</IsTrimmable>` in the repo, so you won't be the first one.

Reply:
If you disable trimming on the assembly then I would remove this file. As per your comment above it's not guaranteed to fix the problem when trimmed anyway (right?), and the user will get warnings.
In the other assemblies which disable trimming by default we don't try to "fix" trimming either (at least as far as I know).